### PR TITLE
dracut/firstboot-network: add default network kargs 

### PR DESCRIPTION
**Pull request by lucab**
This adds the default kargs for firstboot networking, currently mirroring
the ones configured in GRUB.

Ref: https://github.com/coreos/fedora-coreos-tracker/issues/460
Required by: https://github.com/coreos/coreos-assembler/pull/1373
Required by: https://github.com/coreos/afterburn/pull/390

### CODE REVIEW

**Comment by lucab** [comment]
@dustymabe FYI this renames and reuses your dracut module. Is that ok or does this disrupt other things?

**Comment by dustymabe** [comment]
> @dustymabe FYI this renames and reuses your dracut module. Is that ok or does this disrupt other things?

I think that's fine.

File: overlay.d/05core/usr/lib/dracut/modules.d/15coreos-firstboot-network/module-setup.sh [hunk 1]
```diff
@@ -13,4 +13,10 @@ install() {
     # has disks. ignition-diskful.target should suffice.
     install_and_enable_unit "coreos-copy-firstboot-network.service" \
         "ignition-diskful.target"
+
+    # Dropin with firsboot network configuration kargs, applied via
```
Comment (jlebon) [inline overlay.d/05core/usr/lib/dracut/modules.d/15coreos-firstboot-network/module-setup.sh]: `firstboot`

**Comment by lucab** [comment]
Amended.

**Review by jlebon — APPROVED** [review]
LGTM!

**Comment by dustymabe** [comment]
We discussed this in last weeks community meeting. I think we need to hold this until the next round of releases goes out if I recall correctly.

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by lucab** [comment]
~~@dustymabe the one that needs to be hold till the last step is coreos/coreos-assembler#1373 (it's already marked as such). The one here is the first piece to be merged on our path.~~

EDIT: nevermind, we don't **have** to wait for merging this (i.e. it won't break other stuff), but we **decided** to finish the next round of releases first before proceeding.

File: overlay.d/05core/usr/lib/dracut/modules.d/15coreos-firstboot-network/50-afterburn-network-kargs-default.conf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This contains the default kargs for firstboot network configuration.
+# Default values can be dynamically overriden by platform-specific
```
Comment (dustymabe) [inline overlay.d/05core/usr/lib/dracut/modules.d/15coreos-firstboot-network/50-afterburn-network-kargs-default.conf]: typo: -> overridden

**Comment by lucab** [comment]
@dustymabe is there anything else keeping it on hold at this point?

**Comment by lucab** [comment]
Round of releases done, un-holding and merging.

**Comment by dustymabe** [comment]
👍

**Pull request merged**